.github/renovate.json5

```diff
@@ -44,6 +44,7 @@
       ],
       matchStrings: [
         "depName=(?<depName>.*?)\\s.*?_version=(?<currentValue>.*)\\s",
```

Comment:
change the regex to use uppercase VERSION instead of changing the variable names